src/Breadcrumb/BreadcrumbStep.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint react/prop-types: 0 */
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { BreadcrumbStep as _BreadcrumbStep, BreadcrumbLink as _BreadcrumbLink } from './styled';
+
+export type LocalBreadcrumbStepProps = {
+  color?: string;
+  href: string;
```

Comment:
We might want to make this optional if we don't require it to be linked all the time
```suggestion
  href?: string;
```